### PR TITLE
change of address in the link

### PR DESCRIPTION
**Pull request by maretamara**
@marcinwyszynski

### CODE REVIEW

File: templates/index.html [hunk 1]
```diff
@@ -29,7 +29,7 @@ <h1 class="title">
         <h3>Squash it</h3>
         <p> Change your file to this:</p>
         <p class="code">
-          pick f392171 Added new feature X squash ba9dd9a Added new elements to page design squash df71a27 Updated CSS for new elements
```
Comment (marcinwyszynski) [inline templates/index.html:32]: Please don't use `<br />`s.

Comment (maretamara) [inline templates/index.html:32]: why?

Comment (marcinwyszynski) [inline templates/index.html:32]: Because it's mixing markup and presentation.

**Comment by marcinwyszynski** [comment]
OK, please squash commits and feel free to push it.

**Pull request closed without merging**